turtle_graphics/turtle.js

```diff
@@ -0,0 +1,148 @@
+class Turtle {
+    constructor(x, y) {
+        this.x = x;
+        this.y = y;
+        this.currentDirection = "E";
```

Comment:
A good practice to keep code readable and reusable is to never have hard coded values. Set a pointer referencing these values then whenever you need to use the value just reference the pointer. In this case, `"E"` can be turned into a constant.
```javascript
const EAST = "E"
```
a convention is to capitalize all characters in a constant